package.json

```diff
@@ -12,10 +12,10 @@
     "link:emails": "rm -rf emails; ln -s src/generator_templates/ts/emails emails;",
     "link:emails:js": "rm -rf emails; ln -s src/generator_templates/js/emails emails;",
     "build": "preconstruct build",
-    "test": "yarn link:emails; jest && yarn test:next-cli",
-    "test:watch": "yarn test --watch",
-    "test:next-cli": "NEXT_CLI_TESTS=1 jest",
-    "ci": "start-server-and-test ci:server http://localhost:3883 test",
+    "test": "yarn link:emails; yarn jest",
+    "test:next": "yarn link:emails; yarn jest -c jest.next.config.ts",
+    "ci:test": "start-server-and-test ci:server http://localhost:3883 test",
```

Comment:
cool trick, you can just write :3883 as a shorthand for local host:3883 in these